frontend/__tests__/context/configSlice.test.ts

```diff
@@ -50,6 +52,13 @@ describe('config reducer', () => {
     expect(config.dateRange[0].endDate).toEqual('');
   });
 
+  it('should update date range sort status with given sort status', () => {
```

Comment:
Type

Reply:
thanks for reminder, done, no `sortStatus` now